packages/protocol/specs/harnesses/LockedGoldHarness.sol

```diff
@@ -1,10 +1,13 @@
 pragma solidity ^0.5.8;
 
 import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
```

Comment:
these changes are probably harmless but to clarify, these specs are only used in the `certora-test` formal verification and the contracts are never deployed on network afaik

Reply:
thanks, useful information